locales/en/allocations.json

```diff
@@ -37,10 +37,13 @@
       "empty_slot": "Person to be allocated"
     },
     "people_assign": {
+      "special_vehicle_instruction": "You can’t add people who require a special vehicle to an allocation - you need to <a href=\"{{href}}\">create a single request</a> instead",
+      "no_slots_filled": "No prisoners added to allocation",
```

Comment:
What was the outcome of `person` vs `prisoner`? It feels inconsistent to say "Add **person**" on the button but have the text here say "No **prisoners** added...".

Feels like we need to stick to one or the other?

Reply:
`prisoner` was confirmed. Leaving until we raise the issue of content design…